lib/src/int64.dart

```diff
@@ -668,6 +668,14 @@ class Int64 implements IntX {
     return hexStr;
   }
 
+  /**
+   * Returns the digits of [this] when interpreted as an unsigned 64-bit value.
+   */
+  String toStringUnsigned([int radix]) {
```

Comment:
points for using arrow syntax here – since it's a single call

Reply:
I'm just going to keep it consistent with the surrounding code.